content/api-streamer/producer.textile

```diff
@@ -0,0 +1,38 @@
+---
+title: Producer
+section: api-streamer
+index: 67
+---
+
+A Producer of data uses the "API Streamer":https://ably.io/api-streamer to share live data streams with subscribers. Examples of realtime data streams include live sports scores, positioning updates for transport and logistics, or live data streamed from IoT sensors. These data streams can easily be published to the API Streamer via an Ably DSN app, which can be shared with "data consumers":/api-streamer/consumer. 
+
+You can use the API Streamer to "deploy":#deploy, "manage":#manage and "distribute":#distribute your data streams hosted using your DSN apps.
+
+h3(#deploy). Deploy
+
+Built on our globally-distributed, serverless "Data Stream Network":https://www.ably.io/platform, the API Streamer lets you reliably deploy your streaming APIs. Issues typically encountered when deploying realtime APIs such as scalability, bursts of usage, uptime guarantees, message ordering and data integrity, are all taken care of. 
```

Comment:
The things you are talking about are distribution issues, not deploy issues.  I assume @remotekieran for Deploy, we are thinking about deploying data i.e. data flowing from producer to the DSN right?  Then manage is control of those data flows, and distribute is delivery of the data.